.env_sample

```diff
@@ -1,5 +1,4 @@
 export ALLOW_API_ORIGINS=*
```

Comment:
I'm not sure if we use this env?

Reply:
Nope it's a remnant from the previous version where we had an API and a Frontend UI. 👍 
Feel free to remove when you are doing your work. #119